lighthouse-core/config/default-config.js

```diff
@@ -379,6 +388,7 @@ const defaultConfig = {
         {id: 'bootup-time', weight: 0, group: 'diagnostics'},
         {id: 'mainthread-work-breakdown', weight: 0, group: 'diagnostics'},
         {id: 'font-display', weight: 0, group: 'diagnostics'},
+        {id: 'performance-budget', weight: 0, group: 'budgets'},
```

Comment:
```suggestion
        {id: 'performance-budget', weight: 0},
```

the test failures are in the extension and viewer tests--similar to the failures in the report renderer tests that needed the +1 adjustments--where they [test if all the perf audits *with a group* are in the rendered report](https://github.com/GoogleChrome/lighthouse/blob/03e6566fc999befc3e12ca19a3f1bc4b87e8f2d7/lighthouse-viewer/test/viewer-test-pptr.js#L102).

This will be true again after #8708 lands, so it seems like an easy fix for now to remove the group since it doesn't need to be in a group until it's in the report anyways :)